packages/client/src/pages/Destination/DestinationPage.tsx

```diff
@@ -315,6 +351,7 @@ class DestinationPage extends React.Component<DestinationProps, DestinationState
 
 const mapStateToProps = (state: RootState, ownProps: RouteComponentProps<{ placeId: string }>) => {
     return {
+        destinations: state.destinations,
```

Comment:
Just map to `state.destinations.dests` here since you're not using the other store state. 

Or even better, feel free to add another state to destinations store. Call it destination or something, and change destinationReducer to store the result of `getDestinationByPlaceId` in that single destination state instead. This way you don't need to filter and look for that destination in your `getUserRating`.